specification/devcenter/DevCenter/DevBox/models.tsp

```diff
@@ -640,7 +619,7 @@ model DevBoxOperation {
   kind: DevBoxOperationKind;
 
   @doc("The operation status.")
-  status: DevBoxOperationStatus;
+  status: Azure.Core.Foundations.OperationStatus.status;
```

Comment:
You should use `Azure.Core.Foundations.OperationState` instead. You should also update that in [Environments ](https://github.com/Azure/azure-rest-api-specs/blob/64eaae5db602f44416907f25b1fb926cb91e0818/specification/devcenter/DevCenter/Environments/models.tsp#L421) and in [shared](https://github.com/Azure/azure-rest-api-specs/blob/64eaae5db602f44416907f25b1fb926cb91e0818/specification/devcenter/DevCenter/shared/models.tsp#L45)